libs/dh/market-participant/feature-gridarea/src/lib/dh-market-participant-gridarea.component.ts

```diff
@@ -26,53 +26,29 @@ import {
   WattValidationMessageModule,
```

Comment:
suggestion: `WattButtonModule` doesn't look like it's used anymore. Can be removed.